packages/prism-theme-odyssey/webpack.config.js

```diff
@@ -0,0 +1,31 @@
+var path = require('path');
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+
+module.exports = {
+   devServer: {
+    index: 'index.html',
+    contentBase: path.join(__dirname, 'dist'),
+    compress: true,
+    port: 9000
+  },
+  module: {
+    rules: [
+      {
+        test: /\.s[ac]ss$/i,
+        use: [
+          // Creates `style` nodes from JS strings
+          'style-loader',
+          // Translates CSS into CommonJS
+          'css-loader',
+          // Compiles Sass to CSS
+          'sass-loader',
+        ],
```

Comment:
I don't quite follow where are we consuming `sass` and therefore need to use the `sass-loader`. Can you help me understand this better?

Reply:
Answered in #728 714